### PR TITLE
Fix/38 stats total timer

### PR DESCRIPTION
**Pull request by Misarte**

### CODE REVIEW

**Comment by Misarte** [comment]
The conflicts appear on how we display the time on the Text Field, someone coded simultaneously the field to look like: "00:00", while in my code is "0h 0min 0sec". Just let me know which one to keep, I don't mind.

**Comment by Dartemiss** [comment]
I think that hours are not needed and regarding the format 00:00 or 0min 0sec,  I don't know pick whatever you prefer :D

**Comment by Misarte** [comment]
Conflicts resolved guys! Please approve when you have time xD I kept the develop way of displaying time without the hours

File: Scripts/src/Script/WorldManager.cpp [hunk 1]
```diff
@@ -84,12 +84,17 @@ void WorldManager::Awake()
 // Use this for initialization
 void WorldManager::Start()
 {
-
+	//StartTimer();
```
Comment (Dartemiss) [inline Scripts/src/Script/WorldManager.cpp:87]: if not used delete it

File: Scripts/src/Script/WorldManager.cpp [hunk 2]
```diff
@@ -117,7 +122,7 @@ void WorldManager::Update()
 	if(loading && !App->resources->loading_thread_communication.loading)
 	{
 		on_pause = false;
-		StartTimer();
+		//StartTimer();
```
Comment (Dartemiss) [inline Scripts/src/Script/WorldManager.cpp:125]: if not use it delete it

**Review by Dartemiss — APPROVED** [review]

File: Scripts/src/Script/WorldManager.cpp (hunk 1 above)
Comment (Misarte) [reply Scripts/src/Script/WorldManager.cpp]: Fixed!

**Review by yalania — APPROVED** [review]

**Pull request merged**